.golangci.yml

```diff
@@ -1,62 +1,61 @@
+version: "2"
 linters:
-  disable-all: true
+  default: none
   enable:
-    # defaults
-    - govet
     - errcheck
+    - goheader
+    - govet
+    - ineffassign
     - staticcheck
     - unused
-    - gosimple
-    - ineffassign
-    - typecheck
-
-    # custom
-    - goheader
+  settings:
+    goheader:
+      values:
+        regexp:
+          COPYRIGHT: (© Copyright IBM Corporation 20\d{2}(?:, 20\d{2})?\. All Rights Reserved\.|\(C\) Copyright IBM Corp\. 20\d{2}\.)
+      template-path: config/header-template.txt
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    rules:
+      - linters:
+          - errcheck
+          - govet
+          - ineffassign
+          - staticcheck
+          - unused
+        path: cloudantv1/cloudant_v1\.go
+      - linters:
+          - errcheck
+          - govet
+          - ineffassign
+          - staticcheck
+          - unused
+        path: cloudantv1/cloudant_v1_suite_test\.go
```

Comment:
The linters I do not see here however it presents in the previous version are:
       
 - gosimple
- typecheck
- goimports

Reply:
@mojito317 All questions seems to condense to this one, so I'll answer only here if that's all right.

`goimports` is not a linter, so it was moved in `formatters` section in the new config, it is enabled down there.

For the rest quoting the [migration guide](https://golangci-lint.run/product/migration-guide/#linters):
> "typecheck is not a linter, so it cannot be enabled or disabled"
> "The linters stylecheck, gosimple, and staticcheck has been merged inside the staticcheck."

So in short we didn't lose any coverage with this migration.

Reply:
Thanks!